hack/harvester-sb-collector.sh

```diff
@@ -14,22 +14,20 @@ NODE_NAME=${HARVESTER_NODE_NAME:-$(cat ${HOST_PATH}/etc/hostname)}
 BUNDLE_DIR="${OUTPUT_DIR}/${NODE_NAME}"
 
 mkdir -p ${BUNDLE_DIR}
-cd ${BUNDLE_DIR}
-# get some host information
-cp ${HOST_PATH}/etc/hostname .
-
-# collect logs
-mkdir -p logs
-cd logs
-dmesg &> dmesg.log
-
-# k3s logs don't rorate well and can be huge
-tail -c 10m ${HOST_PATH}/var/log/k3s-service.log > k3s-service.log
-tail -c 10m ${HOST_PATH}/var/log/k3s-restarter.log > k3s-restarter.log
-
-cp ${HOST_PATH}/var/log/qemu-ga.log* .
-cp ${HOST_PATH}/var/log/messages* .
-cp ${HOST_PATH}/var/log/console.log .
+
+OS_ID=$(bash -c "source $HOST_PATH/etc/os-release && echo \$ID")
```

Comment:
Note: We need to keep `/etc/os-release` to OpenSUSE for certification reasons. Need to have a discussion with Darren on what should we put/keep there.